sys/include/net/ng_pkt.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2014 Martine Lenders <mlenders@inf.fu-berlin.de>
```

Comment:
Sure we do, also why not start now even if?
https://github.com/RIOT-OS/RIOT/blob/e5e1d0dd6dc81eb52e10e3eba42364f195e8b06c/sys/vtimer/vtimer.c#L4